apps/docs/docs/tutorials/api-sdk/credential-group.md

```diff
@@ -0,0 +1,238 @@
+```
+
+</TabItem>
+</Tabs>
+
+## Create a new instance
+
+After installing the API SDK, create a new instance of `ApiSdk` using the API URL and the [config](https://axios-http.com/docs/req_config).
+
+You can choose to:
+
+-   Create a new instance using the Bandada API URL and the default config.
+
+```ts
+import { ApiSdk } from "@bandada/api-sdk"
+
+const apiSdk = new ApiSdk()
+```
+
+-   Create a new instance using a [Supported URL](https://github.com/bandada-infra/bandada/blob/main/libs/api-sdk/src/types/index.ts#L43).
+
+```ts
+import { ApiSdk, SupportedUrl } from "@bandada/api-sdk"
+
+const apiSdk = new ApiSdk(SupportedUrl.DEV)
+```
+
+## Create a group
+
+### Manual group
+
+Create a new manual group using the API SDK.
+
+```ts
+const apiKey = "your-api-key"
+
+const groupCreationDetails = {
+    name: "Manual Group",
+    description: "This is a description",
+    treeDepth: 16,
+    fingerprintDuration: 3600
+}
+
+const group = apiSdk.createGroup(groupCreationDetails, apiKey)
+```
+
+### Credential group
+
+Create a new credential group using the API SDK.
+
+```ts
+const apiKey = "your-api-key"
+
+const credentials = {
+    id: "BLOCKCHAIN_BALANCE",
+    criteria: {
+        minBalance: "10",
+        network: "Sepolia"
+    }
+}
+
+const groupCreationDetails = {
+    name: "Credential Group",
+    description: "This is a description",
+    treeDepth: 16,
+    fingerprintDuration: 3600,
+    credentials
+}
+
+const credentialGroup = apiSdk.createGroup(groupCreationDetails, apiKey)
+```
+### Multiple credentials group
+
+Create a multiple-credentials group using the API SDK
+
+```ts
+const apiKey = "your-api-key"
+
+const credentials = {
+    credentials: [
+        {
+            id: "BLOCKCHAIN_TRANSACTIONS",
+            criteria: {
+                minTransactions: 10,
+                network: "Sepolia"
+            }
+        },
+        {
+            id: "BLOCKCHAIN_BALANCE",
+            criteria: {
+                minBalance: "5",
+                network: "Sepolia"
+            }
+        }
+    ],
+    expression: ["", "and", ""]
+}
+
+const groupCreationDetails = {
+    name: "Multiple Credentials Group",
+    description: "This is a description",
+    treeDepth: 16,
+    fingerprintDuration: 3600,
+    credentials
+}
+
+const credentialGroup = apiSdk.createGroup(groupCreationDetails, apiKey)
+```
+More details on the credentials can be found at the [Credentials library](https://github.com/bandada-infra/bandada/tree/main/libs/credentials).
+
+## Add members to a group
+
+### Manual group
+
+You can add users to a manual group by using the API SDK.
+
+```ts
+const groupId = "your-group-id"
+const memberId = "member-id-1"
+const apiKey = "your-api-key"
+
+await apiSdk.addMemberByApiKey(
+    groupId,
+    memberId,
+    apiKey
+)
+```
+
+### Credential and multiple credentials group
+
+You can invite users to join a credential and multiple credentials group by generating the credential group join URL.
```

Comment:
```suggestion
You can invite users to join a credential group by generating the credential group join URL.
```

Reply:
This function only works for credential groups. We need to update that function or create a new one to add support to add members to a multiple credential group since the URL is different for both. 

(This will be part of a new issue/PR)

The url for multiple credential groups is: 

```bash
<dashboard-url>/credentials?group=<group-id>&member=<member-id>&type=multiple
```

Example: 

```bash
http://localhost:3001/credentials?group=79609564395766533586300841856737&member=15316025723065452012129624296669307962292967176408402265701167251062457738490&type=multiple
```

More info in the description of this PR: https://github.com/bandada-infra/bandada/pull/530